src/astro/databases/google/bigquery.py

```diff
@@ -34,8 +35,24 @@ def sqlalchemy_engine(self) -> Engine:
 
     @property
     def default_metadata(self) -> Metadata:
+        """
+        Fill in default metadata values for table objects addressing bigquery databases
+        :return:
+        """
         return Metadata(schema=settings.SCHEMA, database=self.hook.project_id)
 
+    def schema_exists(self, schema):
+        """
+        Checks if a dataset exists in the BigQuery
+
+        :param schema: Bigquery namespace
+        """
+        try:
+            self.hook.get_dataset(dataset_id=schema)
+        except GoogleNotFound:
+            return False
+        return True
```

Comment:
Not needed for this PR, but in future sprints we should probably add log lines in all methods, some at DEBUG and some at LOG level so it becomes easier to debug for users when things don't work